### ISSUE TITLE
Thunderbird 128 compatibility?

### ISSUE THREAD

**Issue by metafarion** [LucBennett/BetterUnsubscribe#2]
I'm not actually certain if this is a compatibility issue, but I'm currently running TB 128 and the Unsubscribe button is greyed-out in every email I've tested that includes an unsubscribe link.

**Comment by Rudloff** [comment]
Same here using BetterUnsubscribe 2.6.1 with Thunderbird 128.4.4esr.
I only get this in logs:
```
[BetterUnsubscribe][background.js] Message displayed background.js:9:13
```

**Comment by LucBennett** [comment]
Thanks for reporting this! I’m not aware of any compatibility issues at the moment. For reference, I regularly test BetterUnsubscribe using BetterBird (115.15.0-bb32 (64-bit)), and I haven’t encountered similar behavior so far. That said, I’m looking into it to ensure everything is functioning correctly.

If you’re comfortable sharing, it would be very helpful to receive a (redacted) version of an email where the issue occurs. Specifically, I’d like to confirm whether the unsubscribe link is included in the email headers (via the list-unsubscribe header) or embedded directly in the email content (e.g., as an HTML <a> link or in plaintext).

Thanks again for your help, and I’ll keep you updated on what I find!

**Comment by Rudloff** [comment]
Here is the notification email from your answer as an example: https://gist.github.com/Rudloff/e641dd658beca342924c3f80c6bb5901

It has a `List-Unsubscribe` header but the button stays disabled.

Note that if I disabled/re-enable the plugin while the e-mail is opened (in another tab), the button becomes active and works.
Then if I close and reopen the e-mail, the button is disabled.

**Comment by LucBennett** [comment]
Thank you for reporting this issue and for your patience.

The issue should now be resolved in version **2.7.0**. Please update to the latest version and let me know if the problem persists.

### Details:
After further investigation, it appears there was an undocumented change to the `messenger.messageDisplayAction` module. Specifically, the [optional `tabId` parameter](https://webextension-api.thunderbird.net/en/stable/messageDisplayAction.html#disable-tabid) for enabling and disabling the button turned out to be **required**. 

This behavior was unexpected, as the parameter is documented as optional. By reviewing the source code and testing different scenarios, I was able to adapt the implementation to account for this change. The necessary adjustments have been incorporated in version 2.7.0.

Thank you again for bringing this to my attention!

**Comment by Rudloff** [comment]
I confirm it works with 2.7.0, thanks!
